sermant-agentcore/sermant-agentcore-core/src/main/java/io/sermant/core/service/metric/Metrics.java

```diff
@@ -0,0 +1,312 @@
+/*
+ * Copyright (C) 2024-2024 Huawei Technologies Co., Ltd. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.sermant.core.service.metric;
+
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
+import java.util.function.ToDoubleFunction;
+
+/**
+ * metric
+ *
+ * @author zwmagic
+ * @since 2024-08-17
+ */
+public class Metrics {
```

Comment:
The name must be specific, for example, MetricManager.

Reply:
these are same field definitions as micrmeter which are wrappered to use in Sermant